inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.java

```diff
@@ -931,4 +955,19 @@ else if (vMarkerType.equals(TEXT_MARKER)) {
             }
         }
     }
+
+    private void setUserState()
```

Comment:
All components must directly read/write the AL state fields. So this method should be removed and proper models (e.g. LambdaModels)  should be used which connect components directly to the underlying state.